thoth/storages/graph/postgres.py

```diff
@@ -406,6 +406,26 @@ def get_alembic_version_count_all(self) -> int:
             result = session.execute(query).fetchone()
             return result[0]
 
+    def get_last_solver_datetime(
+        self, os_name: Optional[str] = None, os_version: Optional[str] = None, python_version: Optional[str] = None
+    ) -> datetime:
+        """Get the datetime of the last solver run synced in the database."""
+        with self._session_scope() as session:
+            result = session.query(Solved.datetime)
+            if os_name or os_version or python_version:
+                result = result.filter(Solved.ecosystem_solver_id == EcosystemSolver.id)
+
+                if os_name is not None:
+                    result = result.filter(EcosystemSolver.os_name == os_name)
+
+                if os_version is not None:
+                    result = result.filter(EcosystemSolver.os_version == os_version)
+
+                if python_version is not None:
+                    result = result.filter(EcosystemSolver.python_version == python_version)
+
+        return datetime.strptime(max([datetime[0].isoformat() for datetime in result.all()]), "%Y-%m-%d %H:%M:%S.%f")
```

Comment:
`ValueError: time data '2022-02-06T23:08:06.688200' does not match format '%Y-%m-%d %H:%M:%S.%f'`